testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/deployment/DeploymentScannerShutdownTestCase.java

```diff
@@ -120,22 +110,32 @@ public static void addDeploymentScanner() throws Exception {
 
     @After
     public void cleanAll() throws Exception {
-        removeRules();
-        removeDeploymentScanner(client, scannerName);
-        removeDeploymentScannerExtension();
+        try {
+            removeRules();
+        } catch (Exception e) {
+
+        }
+        try {
+            removeDeploymentScanner(scannerName);
+        } catch (Exception e) {
+
+        }
+        try {
+            removeDeploymentScannerExtension();
+        } catch (Exception e) {
+
+        }
```

Comment:
not really, they are here just so that container.stop() is called for sure.
otherwise we would get inconsistent state, they also log all the errors.